charts/spinnaker/README.md

```diff
@@ -81,9 +81,9 @@ kubeConfig:
   contexts:
   # Names of contexts available in the uploaded kubeconfig
   - my-context
-  # This is the context from the list above that you would like
-  # to deploy Spinnaker itself to.
-  deploymentContext: my-context
+  # This is the Spinnaker account that you would like to use to
+  # deploy Spinnaker itself with.
+  deploymentAccount: my-context
```

Comment:
So in this case, the account matches the context name. Changing this would require changing it under `contexts` too, which is odd. We can make it more 'real' maybe by making it e.g. `production`. WDYT?

Reply:
Yeah it SGTM man 😄.